.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Personally, I don't like the idea of suggesting extensions for a few reasons:

1. Some people don't use VSCode
2. Those who do may not want to install extensions
3. I don't personally think code should rely/require/suggest extensions. That's up to the user to decide.

Reply:
extensions are opt-in. People can completely ignore that even if they are using VSCode. Plus I got positive comments on listing suggested extensions. For many starters they are valuable. Boilerplate is already opinionated in everything. Suggestions is the least opinionated part I guess